.travis.yml

```diff
@@ -0,0 +1,82 @@
+language: cpp
+
+# against hwloc error (travis-ci/travis-ci#10019)
+sudo: required
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - ccache
+      - libhwloc-dev
+      - libopenmpi-dev
+      - openmpi-bin
+      - g++-6
+
+before_script:
+  - sudo ln -s /usr/bin/ccache /usr/lib/ccache/clang++
+  - ccache -z
+  - KOKKOS_OPTS=( --with-hwloc=/usr --gcc-toolchain=/usr --with-serial --with-openmp)
+    # LD_LIBRARY_PATH workaround for libomp: https://github.com/travis-ci/travis-ci/issues/8613
+  - if [[ ${CC} = clang ]]; then export LD_LIBRARY_PATH=/usr/local/clang/lib:$LD_LIBRARY_PATH; fi
+  - if [[ ${CC} = gcc ]]; then export CXX=g++-6; export CC=gcc-6; export FC=gfortran-6; fi
+  - git clone --depth=1 https://github.com/kokkos/kokkos.git &&
+    pushd kokkos &&
+    mkdir build &&
+    pushd build &&
+    ../generate_makefile.bash --prefix=$HOME/kokkos ${KOKKOS_OPTS[@]} &&
+    make -j2 &&
+    make install &&
+    popd &&
+    popd
+  - pwd
+  - git clone --depth=1 https://github.com/ECP-copa/Cabana.git
+  - pushd Cabana && mkdir build && pushd build &&
+    cmake -DCMAKE_PREFIX_PATH=$HOME/kokkos
+          -DCMAKE_INSTALL_PREFIX=$HOME/Cabana
+          -DCabana_ENABLE_Serial=ON -DDCabana_ENABLE_OpenMP=ON
```

Comment:
`-DDCabana_ENABLE_OpenMP` has an extra `D`

Reply:
@rfbird since I stole this from CabanaPIC, that's hiding there as well

Reply:
Thanks for pointing this out. I thought I lifted this from the Cabana repo, but It looks fine over there.